tests/unit/test_utils.py

```diff
@@ -304,6 +304,13 @@ def test_get_temp_filename(self):
                 OSUtils().get_temp_filename(filename)
             )
         )
+    def test_get_temp_filename_len_255(self):
```

Comment:
Let's add a newline above this. Functions inside of a class should be separated by a single new line.